.travis.yml

```diff
@@ -20,6 +20,7 @@ env:
 matrix:
   fast_finish: true
   allow_failures:
+    - env: EMBER_TRY_SCENARIO=ember-beta
```

Comment:
@hhff Looks like you're aware that there are some ember 2.0 issues (#32). For now I think you guys should allow test failures in "beta", and then once you get the 2.0 issues fixed, stop allowing those failures